src/data/vector/basic.lean

```diff
@@ -55,13 +55,23 @@ instance zero_subsingleton : subsingleton (vector α 0) :=
   ∀ (v : vector α n) h, (⟨to_list v, h⟩ : vector α n) = v
 | ⟨l, h₁⟩ h₂ := rfl
 
+@[simp]
+lemma length_coe (v : vector α n) :
+  ((coe : { l : list α // l.length = n } → list α) v).length = n :=
+v.2
+
 @[simp] lemma to_list_map {β : Type*} (v : vector α n) (f : α → β) : (v.map f).to_list =
   v.to_list.map f := by cases v; refl
 
 theorem nth_eq_nth_le : ∀ (v : vector α n) (i),
   nth v i = v.to_list.nth_le i.1 (by rw to_list_length; exact i.2)
 | ⟨l, h⟩ i := rfl
 
+@[simp]
+lemma repeat_nth (a : α) (i : fin n) :
```

Comment:
```suggestion
lemma nth_repeat (a : α) (i : fin n) :
```

Reply:
This seems to be what matches the naming convention of the lemmas surrounding it.